src/core/components/AppAddrQr.vue

```diff
@@ -15,7 +15,7 @@
     =====================================================================================
     -->
     <div class="textPrimary--text mb-8">
-      {{ $t('interface.qr.desc') }}
+      {{ $t('interface.qr.desc').replace(/ETH/g, currency) }}
```

Comment:
use this instead: https://kazupon.github.io/vue-i18n/guide/interpolation.html#slots-syntax-usage